src/Writer/XLSX/Manager/WorksheetManager.php

```diff
@@ -142,6 +149,56 @@ private function addNonEmptyRow(Worksheet $worksheet, Row $row): void
         }
     }
 
+    /**
+     * Adds empty row to the worksheet.
+     *
+     * @param Worksheet $worksheet The worksheet to add the row to
+     * @param Row       $row       The row to be written
+     *
+     * @throws InvalidArgumentException If a cell value's type is not supported
+     * @throws IOException              If the data cannot be written
+     */
+    private function addEmptyRow(Worksheet $worksheet, Row $row): void
+    {
+        $sheetFilePointer = $worksheet->getFilePointer();
+        $rowIndexOneBased = $worksheet->getLastWrittenRowIndex() + 1;
+        $rowAttributes = ''
+            ." r=\"{$rowIndexOneBased}\""
+            .$this->getRowAttributes($row);
+
+        $rowXML = "<row{$rowAttributes}></row>";
+
+        $wasWriteSuccessful = fwrite($sheetFilePointer, $rowXML);
+        if (false === $wasWriteSuccessful) {
+            throw new IOException("Unable to write data in {$worksheet->getFilePath()}");
+        }
```

Comment:
I see you've just copy-pasted this and can't be tested: I say let's just assert it:
```suggestion
        assert(false !== $wasWriteSuccessful);
```